secrets/vault/vault.go

```diff
@@ -96,6 +96,11 @@ func (k *keeper) Encrypt(ctx context.Context, plaintext []byte) ([]byte, error)
 	return []byte(secret.Data["ciphertext"].(string)), nil
 }
 
+// ErrorAs implements driver.Keeper.ErrorAs.
```

Comment:
Same here, add to package docstring even if it doesn't support any types.

What error type does `Vault` return? Maybe add to the `TODO` below, something like "classify vault error codes and add a type for ErrorAs".

Reply:
Done.